Composer/packages/client/src/CreationFlow/index.js

```diff
@@ -21,8 +21,27 @@ export function CreationFlow(props) {
   const [step, setStep] = useState();
   // eslint-disable-next-line react/prop-types
   const { creationFlowStatus, setCreationFlowStatus } = props;
-  const { fetchTemplates, openBotProject, createProject, saveProjectAs, saveTemplateId, fetchStorages } = actions;
-  const { templateId, templateProjects, focusedStorageFolder } = state;
+  const {
+    fetchTemplates,
+    openBotProject,
+    createProject,
+    saveProjectAs,
+    saveTemplateId,
+    fetchStorages,
+    fetchFolderItemsByPath,
+  } = actions;
+  const { templateId, templateProjects, focusedStorageFolder, storages } = state;
+  const currentStorageIndex = useRef(0);
+  const currentStorageId = storages[currentStorageIndex.current] ? storages[currentStorageIndex.current].id : 'default';
+  const [currentPath, setCurrentPath] = useState('');
+  useEffect(() => {
+    if (storages && storages.length > 0) {
+      const storageId = storages[currentStorageIndex.current].id;
+      const path = storages[currentStorageIndex.current].path;
+      const formatedPath = Path.normalize(path);
+      fetchFolderItemsByPath(storageId, formatedPath);
+    }
+  }, [storages]);
```

Comment:
As `currentStorageIndex` is a ref, if it get changed, it will not run `fetchFolderItemsByPath`.

And also, I scanned the code and it seems `currentStorageIndex` never get changed, it is always `0`.